code/obj/machinery/plantpot.dm

```diff
@@ -99,6 +99,7 @@
 		// We have this here as a check for whether or not the plant needs to update its sprite.
 		// Originally plantpots updated constantly but this was found to be rather expensive, so
 		// now it only does that if it needs to.
+	var/actionpassed //variable for action bar harvesting yay :D 0 = no action, 1 = action passed, 2 = action cancelled
```

Comment:
please make use of #defines here for the values

Reply:
Sorry for the delay, I'll figure out how to do that!

Reply:
Alright, I think I figure it out.